scripts/gitlab/check_extrinsics_ordering.sh

```diff
@@ -32,19 +46,17 @@ for RUNTIME in "${runtimes[@]}"; do
     exit 0
   fi
 
-  if [ "$RUNTIME" = 'polkadot' ]; then
-    LIVE_WS="wss://rpc.polkadot.io"
-  else
-    LIVE_WS="wss://${RUNTIME}-rpc.polkadot.io"
-  fi
-
-  # Start running the local polkadot node in the background
-  $BIN --chain="$RUNTIME-local"  &
+  # Start running the nodes in the background
+  $HEAD_BIN --chain="$RUNTIME-local" --tmp &
+  $RELEASE_BIN --chain="$RUNTIME-local" --ws-port 9945 --tmp &
   jobs
 
+  # Sleep a little to allow the nodes to spin up and start listening
+  sleep 5
```

Comment:
What happens when it takes more than 5 seconds?

Reply:
maybe it makes sense to wait for a certain reaction from a node.

Reply:
Good point. Added a check for listening ports that times out after X seconds (currently 5). In reality, if the nodes are working correctly they'll usually start listening after < 1s but this'll also pick up if something's wrong with the binary (though one would hope that would have been detected in test-linux-stable)